cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
any need of this? we want to test against an actual instance without mocking for e2e tests

Reply:
Sorry, sorry I didn't create that.

Reply:
I'll remove it.